### DISCUSSION TITLE
VDH breaks clipboard

### DISCUSSION THREAD

**Discussion by mpopp75** [aclap-dev/video-downloadhelper#1196] [Help and Q&A]
### Your question

Using "Download as...", if I copy some text into the clipboard (e.g. the date to add it to every video), when I download the next video, the clipboard content is gone.

Using Xubuntu 20.04 and Firefox 130.0.

### Addon & CoApp version and details

version: 9.1.0.42
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"/opt/vdhcoapp/vdhcoapp","version":"2.0.19","new_version":false}
license: {"unneeded":true}
platform: x86-64 linux
UA: Mozilla/5.0 (X11; Linux x86_64; rv:130.0) Gecko/20100101 Firefox/130.0
{
    "download_directory": "/home/mpopp/Downloads",
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive"
        }
    },
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": false,
        "show_button_clean_all": true,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "default_action": "download_as",
    "open_count_store": 11,
    "successfull_dl": 4
}

**Comment by Mattmjs** [comment]
More likely it's your system, I tried that and did two downloads and it still remembered what was in system clipboard
on Windows 10.

**Reply by GrampaWildWilly** [reply]
I've never seen this interaction, either, on Windows 11.  Must be an issue unique to Xubuntu.

**Comment by mpopp75** [comment]
Even if it's unique to Xubuntu, it's still a bug.

**Reply by GrampaWildWilly** [reply]
But is the bug in VDH or Xubuntu?  I'm certainly not in a position to even investigate this.  Not only do I not work for VDH, so I don't have access to the code, which I wouldn't understand anyway, but I know diddly squat about any flavor of Linux.  I do know that Xubuntu is a flavor of Linux.  But my knowledge stops there.  The fact that the problem does not occur on Windows strongly points to the culprit being Xubuntu & not VDH.  But I would defer to anything @paulrouget Paul would say on the subject.  Unlike me, he actually is the developer.  Let's hope he looks in here & has something helpful to offer.
